docs/shared_parsers_catalog/system_time.rst

```diff
@@ -1,3 +1,38 @@
-.. automodule:: insights.parsers.system_time
+System time configuration
+=========================
+
+file ``/etc/chrony.conf``
+-------------------------
+.. autoclass:: insights.parsers.system_time.ChronyConf
+   :members:
+   :show-inheritance:
+
+file ``/etc/sysconfig/chronyd``
+-------------------------------
+.. autoclass:: insights.parsers.system_time.ChronydService
+   :members:
+   :show-inheritance:
+
+command ``file -L /etc/localtime``
+----------------------------------
+.. autoclass:: insights.parsers.system_time.LocalTime
+   :members:
+   :show-inheritance:
+
+file ``/etc/ntp.conf``
+----------------------
+.. autoclass:: insights.parsers.system_time.NTP_conf
+   :members:
+   :show-inheritance:
+
+command ``ntptime``
+-------------------
+.. autoclass:: insights.parsers.system_time.NtpTime
+   :members:
+   :show-inheritance:
+
+file ``/etc/sysconfig/ntpd``
+----------------------------
+.. autoclass:: insights.parsers.system_time.NTPDService
```

Comment:
@PaulWay it does need to be included in the documentation so that it will be linked from the child class and users will be able to understand implementation of the child class.